packages/instantsearch.js/src/widgets/chat/chat.tsx

```diff
@@ -197,6 +197,7 @@ const createRenderer = <THit extends NonNullable<object> = BaseHit>({
 
       const [isClearing, setIsClearing] = state.use(false);
       const [maximized, setMaximized] = state.use(false);
+      const [promptRef] = state.use({ current: null });
```

Comment:
Do you think the local state is useful for other widgets too, like refinement list show more? would be good in some follow up later to make those patterns equal.

Reply:
I suppose we could expose the `open`/`setOpen` local state (and others) as props (`open`/`onToggleOpen`) while keeping them in an uncontrolled mode by default. This would allow for customization while keeping the usage the same, matching the other widgets.

WDYT?

Reply:
Not sure what you mean by that to be honest. how can you have an open/setOpen prop in JS?